lib/constants/constants.go

```diff
@@ -98,4 +102,12 @@ const (
 var (
 	// GravityDataDir is the directory where gravity data is stored in planet
 	GravityDataDir = "/var/lib/gravity"
+
+	// MinKernelVersion is the minimum supported Linux kernel version -- 3.10.0-1127
```

Comment:
Nit: so this is maybe a bit miss-leading, and it's an artifact of the original bug report from @r0mant . We need to be careful here because all kernels aren't equal. So this check probably only works in this particular case, because I image redhat releases which is what this is meant to protect against are running the oldest kernels. But it's unsafe to see say an ubuntu kernel bug and simply increase this value.

So I would try and atleast document this in the comment to a further extent, to try and avoid someone accidentally using this and triggering some unintended side effects between the various distros and the in use kernels.